modules/rational-ui.el

```diff
@@ -55,18 +55,63 @@ Use a plist with the same key names as accepted by `set-face-attribute'.")
 
 ;;;; Line Numbers
 
+(defcustom rational-ui-line-numbers-enabled-modes
+  '(conf-mode prog-mode)
+  "Modes which should display line numbers."
+  :type 'list
+  :group 'rational)
+
+(defcustom rational-ui-line-numbers-disabled-modes
+  '(org-mode)
+  "Modes which should not display line numbers.
+Modes deriverd from the modes defined in
```

Comment:
```suggestion
Modes derived from the modes defined in
```

Reply:
2nd one of these... I obviously cannot spell.